LeaderboardBackend/Services/Impl/AccountConfirmationService.cs

```diff
@@ -48,6 +49,7 @@ await emailSender.EnqueueEmailAsync(
         }
         catch
         {
+            logger.LogWarning("Account confirmation email failed to send");
```

Comment:
[Optional] Include the user ID in this message for troubleshooting purposes. That way when we're looking at errors, we can tell whether or not it's happening for multiple users.